tests/ml/test_predictor_ml.py

```diff
@@ -81,6 +81,7 @@ def test_generate_compiled_circuits():
     qasm_path = Path("compiled_test.qasm")
     qc.qasm(filename=str(qasm_path))
     predictor.generate_compiled_circuits(source_path, str(target_path))
+    assert len(list(target_path.glob("*.qasm"))) > 0
```

Comment:
Just as above, this can be made a bit more performant
```suggestion
    assert any(file.suffix == ".qasm" for file in target_path.iterdir())
```

Reply:
Done